src/lib/functions/synchronous.js

```diff
@@ -36,14 +38,36 @@ const handleSynchronousFunction = function (err, result, response) {
   response.end()
 }
 
-const formatLambdaLocalError = (err) => `${err.errorType}: ${err.errorMessage}\n  ${err.stackTrace.join('\n  ')}`
+const formatLambdaLocalError = (err, acceptsHtml) =>
+  acceptsHtml
+    ? JSON.stringify({
+        errorType: err.errorType,
+        errorMessage: err.errorMessage,
+        trace: err.stackTrace,
+      })
+    : `${err.errorType}: ${err.errorMessage}\n ${err.stackTrace.join('\n ')}`
 
-const handleErr = function (err, response) {
+const renderErrorTemplate = async (errString) => {
+  const regexPattern = /<!--@ERROR-DETAILS-->/g
+  let errorTemplateFile
```

Comment:
Since you're declaring `errorTemplateFile` in the scope of `renderErrorTemplate`, it will always be falsy when you evaluate it in L54.

I think you want to move this declaration out of `renderErrorTemplate`?